packages/fresh_graphql/pubspec.yaml

```diff
@@ -7,13 +7,13 @@ homepage: https://github.com/felangel/fresh
 version: 0.5.2
 
 environment:
-  sdk: ">=2.12.0 <3.0.0"
+  sdk: ">=2.19.0 <3.0.0"
 
 dependencies:
   fresh: ^0.4.0
-  graphql: ^5.0.0
-  http: ^0.13.1
+  graphql: ^5.1.3
```

Comment:
Can we resolve the merge conflicts since fresh_graphql no longer depends on package:graphql? Thanks!